packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -675,6 +675,9 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
   ScrollActivity get activity => _activity;
   ScrollActivity _activity;
 
+  /// The current [ScrollActivity.velocity] of the current activity, if any.
+  double get currentScrollVelocity => activity?.velocity;
```

Comment:
Document units in docstring and/or property name.

Reply:
This is still outstanding =)

Reply:
Hm.  I thought I did this one, I think I did the similar one instead.  Will update.

Reply:
GitHub is messed up here.  The text acutally reads:

```
  ///
  /// This value is in logical pixels per second. It may be positive, negative,
  /// 0, or null.
```

In addition to what's showing right now.  You can see it at: https://github.com/flutter/flutter/pull/48536/commits/f732fbe674d87ac557a9eb123a694aeae0da71d7

Reply:
Cool thanks, what's a velocity of null mean?

Reply:
Hm.  I might be wrong about this.  I was thinking there was some code path somewhere that could set it to null or set the activity to null.  I'm having trouble finding it.  @goderbauer may know if this is accurate.

Reply:
Oh, derp.

This can return null if you're not in a scrollable context :)

Reply:
It's not clear to me why this should return null.

> This can return null if you're not in a scrollable context :)

How would you have a ScrollPosition if you're not in a scrollable context?

Also: What does it mean from an API point of view when your velocity is null? Shouldn't this always return negative, 0 or positive numbers?